sdk/core/azure-core/setup.py

```diff
@@ -42,14 +42,13 @@
     license='MIT License',
     author='Microsoft Corporation',
     author_email='azpysdkhelp@microsoft.com',
-    url='https://github.com/Azure/azure-sdk-for-python',
+    url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/core/azure-core',
     classifiers=[
         'Development Status :: 4 - Beta',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
```

Comment:
I notice other services `setup.py` files still list Python 3.4 in their classifiers, just curious if there is a plan or governing issue to bring all of these into consistency?

Reply:
I confirmed with Johan (he updated guideline too), we will not support 3.4